test/Interop/SwiftToC/functions/swift-primitive-functions-c-bridging.swift

```diff
@@ -4,7 +4,7 @@
 
 // RUN: %check-interop-c-header-in-clang(%t/functions.h)
 
-// CHECK: SWIFT_EXTERN float $s9Functions16passThrougCFloatyS2fF(float x) SWIFT_NOEXCEPT SWIFT_CALL;
```

Comment:
[CI test failure](https://ci.swift.org/job/swift-PR-Linux/16278/consoleFull#-453661406d6fdb6cb-f376-4f2e-8bce-d31c7304698b)

I suspect because `passThroughCFloat` (changed from `passThrougCFloat`) will have a different... what is that identfier? mangled? I lack the correct terminology... how can I know which is the correct one?

Reply:
A mangled name, yeah. As you may have figured, you forgot to update the encoded length of the function name. In the general case, running the test and inspecting the output often helps. Otherwise, you can grab the part of the expanded test command that is piped to FileCheck from that same output, run it yourself, and find the mangled name by source name.